volatility3/framework/plugins/linux/modxview.py

```diff
@@ -0,0 +1,195 @@
+        """
+        seen_addresses = set()
+        for modules in run_results.values():
+            for module in modules:
+                if deduplicate and module.vol.offset in seen_addresses:
+                    continue
+                yield module
+
+    @classmethod
+    def run_modules_scanners(
+        cls,
+        context: interfaces.context.ContextInterface,
+        kernel_name: str,
+        run_hidden_modules: bool = True,
+    ) -> Dict[str, List[extensions.module]]:
+        """Run module scanning plugins and aggregate the results.
+
+        Args:
+            run_hidden_modules: specify if the hidden_modules plugin should be run
+        Returns:
+            Dictionary mapping each plugin to its corresponding result
+        """
+
+        kernel = context.modules[kernel_name]
+        run_results = {}
+        run_results["lsmod"] = cls.run_lsmod(context, kernel_name)
+        run_results["check_modules"] = cls.run_check_modules(context, kernel_name)
+        if run_hidden_modules:
+            known_module_addresses = set(
+                context.layers[kernel.layer_name].canonicalize(module.vol.offset)
+                for module in run_results["lsmod"] + run_results["check_modules"]
+            )
+            run_results["hidden_modules"] = cls.run_hidden_modules(
+                context, kernel_name, known_module_addresses
+            )
+
+        return run_results
+
+    def _generator(self):
+        kernel_name = self.config["kernel"]
+        run_results = self.run_modules_scanners(self.context, kernel_name)
+        modules_offsets = {}
+        for key in ["lsmod", "check_modules", "hidden_modules"]:
+            modules_offsets[key] = set(module.vol.offset for module in run_results[key])
+
+        seen_addresses = set()
+        for modules_list in run_results.values():
+            for module in modules_list:
+                if module.vol.offset in seen_addresses:
+                    continue
+                seen_addresses.add(module.vol.offset)
+
+                if self.config.get("plain_taints"):
+                    taints = module.get_taints_as_plain_string()
+                else:
+                    taints = ",".join(module.get_taints_parsed())
+
+                yield (
+                    0,
+                    (
+                        module.get_name() or NotAvailableValue(),
+                        format_hints.Hex(module.vol.offset),
+                        module.vol.offset in modules_offsets["lsmod"],
+                        module.vol.offset in modules_offsets["check_modules"],
+                        module.vol.offset in modules_offsets["hidden_modules"],
+                        taints or NotAvailableValue(),
+                    ),
+                )
+
+    def run(self):
+        columns = [
+            ("Name", str),
+            ("Address", format_hints.Hex),
+            ("In /proc/modules", bool),
+            ("In /sys/module/", bool),
```

Comment:
It should work fine, but IMO it's better to avoid using "/" in the column name to make filtering by column easier later on. Maybe `procfs` and `sysfs` or similar?